src/data/ACTIONS/root/BLU.ts

```diff
@@ -14,6 +14,8 @@ export const BLU_COOLDOWN_GROUP = {
 	NIGHTBLOOM: 23290,
 	THE_ROSE_OF_DESTRUCTION: 23275,
 	MATRA_MAGIC: 23285,
+	MAGIC_HAMMER: 18305,
```

Comment:
Minor nitpick I didn't pick up on before. Can we call this const BLU_COOLDOWN_GROUPS to better reflect what it's defining?

Reply:
Done!